api/python/cell_census/src/cell_census/_experiment.py

```diff
@@ -1,33 +1,42 @@
+# Copyright (c) 2022-2023 Chan Zuckerberg Initiative
+#
+# Licensed under the MIT License.
+
+"""
+Experiments handler.
+
+Contains methods to retrieve SOMA Experiments.
+"""
+
 import re
 
 import tiledbsoma as soma
 
 
 def _get_experiment(census: soma.Collection, organism: str) -> soma.Experiment:
-    """
-    Given a census ``soma.Collection``, return the experiment for the named organism.
+    """Given a census ``soma.Collection``, return the experiment for the named organism.
     Organism matching is somewhat flexible, attempting to map from human-friendly
-    names to the underlying collection element name.  Will raise a ``ValueError`` if
-    unable to find the specified organism [lifecycle: experimental].
+    names to the underlying collection element name.
+
+    [lifecycle: experimental]
+
+    Args:
+        census: soma.Collection, The census
+        organism: str, The organism name, eg., ``Homo sapiens``
+
+    Returns:
+        A soma.Experiment object with the requested experiment.
 
-    Parameters
-    ----------
-    census - ``soma.Collection``
-        The census
-    organism - ``str``
-        The organism name, eg., ``Homo sapiens``
+    Raises:
+        ValueError: if unable to find the specified organism
 
-    Returns
-    -------
-    ``soma.Experiment`` - the requested experiment.
+    Examples:
 
-    Examples
-    --------
-    >>> human = get_experiment(census, 'homo sapiens')
+        >>> human = get_experiment(census, 'homo sapiens')
 
```

Comment:
some examples (e.g., this file) have an empty line between them, some (e.g., get_anndata) do not.  I like the extra whitespace, but please use your own discretion about which we should adopt.